docs/tokens/token_list.md

```diff
@@ -24,43 +24,28 @@ In addition to NEON and wNEON tokens, below are some common tokens that are depl
 
 ---
 
-|Token|Mainnet Address                           |
+|Token|Mainnet ERC20 Wrapper Address                        |
 |-----|:-----------------------------------------|
-|USDC |Comming soon...                           |
-|USDT |Comming soon...                           |
-|wSOL |Comming soon...                           |
+|NEON| 0x34260f162a1CcB3ad6c2D974d8e330f54b2c5345|
+|USDC |0x512E48836Cd42F3eB6f50CEd9ffD81E0a7F15103                          |
+|USDT |0x6eEf939FC6e2B3F440dCbB72Ea81Cd63B5a519A5                           |
+|wSOL |0xc7Fc9b46e479c5Cb42f6C458D1881e55E6B7986c                           |
```

Comment:
Also -- this "token" is confusing 
you put in NEON and get out wNEON seems logical
you put in wSOL and get out what wwSol ?? -- does not seem logical -- should that entry on line 32 be SOL rather than wSOL ?? @anton-lisanin

Reply:
NEON and wNEON are completely different and not related to each other.
wNEON should not be on this page
 

It should be wSOL.
And _wrapping_ is not correct word for all of this

Reply:
TODO: add separate page about wNEON

Reply:
@anton-lisanin So the issue is that the w started with the likes of [wETH](https://www.bitcoin.com/get-started/what-is-weth/#:~:text=WETH%20stands%20for%20Wrapped%20Ether,with%20general%20purpose%20smart%20contracts.)

so -- w typically means wrapped 

Now we are trying to use the w to mean ..... ????